package.json

```diff
@@ -78,6 +78,7 @@
     "@spalger/test-subj-selector": "0.2.1",
     "@spalger/ui-ace": "0.2.3",
     "JSONStream": "1.1.1",
+    "accept-language-parser": "1.1.2",
```

Comment:
this needs to be `@1.2.0`  to fix https://github.com/opentable/accept-language-parser/issues/8